lib/foreman_maintain/concerns/base_database.rb

```diff
@@ -2,7 +2,7 @@ module ForemanMaintain
   module Concerns
     module BaseDatabase
       def data_dir
-        if el7? && check_min_version('foreman', '2.0')
+        if el7? && check_min_version('foreman-proxy', '2.0')
```

Comment:
I think instead checking version of foreman-proxy is good?

Reply:
Latest change checks for `rh-postgresql12-postgresql-server-syspaths` instead of `foreman`/`foreman-proxy`